pkg/manager/impl/validation/validation.go

```diff
@@ -121,6 +145,42 @@ func ValidateNamedEntityIdentifierListRequest(request admin.NamedEntityIdentifie
 	return nil
 }
 
+func ValidateNamedEntityGetRequest(request admin.NamedEntityGetRequest) error {
```

Comment:
in following with other specific validations, can you put this in its own file e.g. https://github.com/lyft/flyteadmin/blob/master/pkg/manager/impl/validation/execution_validator.go

Reply:
Yep, done.